### ISSUE TITLE
use TENxMatrix hasn't  feature name

### ISSUE THREAD

**Issue by Liripo** [Bioconductor/HDF5Array#49]
Hello Developer :
My file is as follows,
```R
> h5ls("./filtered_feature_bc_matrix.h5")
              group          name       otype  dclass      dim
0                 /        matrix   H5I_GROUP                 
1           /matrix      barcodes H5I_DATASET  STRING    11952
2           /matrix          data H5I_DATASET INTEGER 27114202
3           /matrix      features   H5I_GROUP                 
4  /matrix/features _all_tag_keys H5I_DATASET  STRING        1
5  /matrix/features  feature_type H5I_DATASET  STRING    36601
6  /matrix/features        genome H5I_DATASET  STRING    36601
7  /matrix/features            id H5I_DATASET  STRING    36601
8  /matrix/features          name H5I_DATASET  STRING    36601
9           /matrix       indices H5I_DATASET INTEGER 27114202
10          /matrix        indptr H5I_DATASET INTEGER    11953
11          /matrix         shape H5I_DATASET INTEGER        2
```
When i read use `TENxMatrix`,the feature name is null: 

```R
> test <- TENxMatrix("./filtered_feature_bc_matrix.h5",group = "matrix")
> test
<36601 x 11952> sparse matrix of class TENxMatrix and type "integer":
         AAACCCAAGGCCCAAA-1 ... TTTGTTGTCTCATTAC-1
    [1,]                  0   .                  0
    [2,]                  0   .                  0
    [3,]                  0   .                  0
    [4,]                  0   .                  0
    [5,]                  0   .                  0
     ...                  .   .                  .
[36597,]                  0   .                  0
[36598,]                  0   .                  0
[36599,]                  0   .                  0
[36600,]                  0   .                  0
[36601,]                  0   .                  0
```

**Comment by LiNk-NY** [comment]
Hi @Liripo 
The `TENxMatrix` function does not add them by default.
You could add them manually by extracting the feature data from the H5 file. 
I have an experimental package at https://github.com/LiNk-NY/TENxIO that you could try. 
Best,
Marcel

**Comment by Liripo** [comment]
@LiNk-NY ：
Thanks for the advice and I will try your experimental package.
Best
liripo!

**Comment by hpages** [comment]
@Liripo @LiNk-NY Should be fixed in **HDF5Array** 1.24.2 (release) and 1.25.2 (devel).

Each new version of **HDF5Array** should become available via `BiocManager::install()` in the next 24-48 hours for BioC 3.15 and BioC 3.16 users, respectively.
